src/onegov/org/cli.py

```diff
@@ -884,3 +889,63 @@ def mark_as_published(request: 'OrgRequest', app: 'OrgApp') -> None:
             )
 
     return mark_as_published
+
+
+@cli.command(name="delete-invisible-links")
+def delete_invisible_links() -> 'Callable[[OrgRequest, OrgApp], None]':
+    """ Deletes all the data associated with a period, including:
+    Example:
+        onegov-org --select /foo/bar delete-invisible-links
+    """
+
+    def delete_invisible_links(request: 'OrgRequest', app: 'OrgApp') -> None:
+        session = request.session
+        query = QueryChain(
+            (session.query(Page),
+             session.query(Resource),
+             session.query(ExtendedDirectory),
+             session.query(FormDefinition))
+        )  # type:ignore
+        models = query.all()
+
+        click.echo(click.style(
+            {session.info["schema"]},
+            fg='yellow'
+        ))
+
+        invisible_links = []
+        for page in models:
+            # Find links with no text, only br tags and/or whitespaces
+            for field in page.content_fields_containing_links_to_files:
+                if not page.content.get(field):
+                    continue
+                soup = BeautifulSoup(page.content.get(field), 'html.parser')
+                for link in soup.find_all('a'):
+                    if not any(
+                        tag.name != 'br' and (
+                            tag.name or not tag.isspace()
+                        ) for tag in link.contents
+                    ):
+                        invisible_links.append(link)
+                        click.echo(click.style(
+                            f"Page: {page.title}", fg='green'))
+                        click.echo(f"Deleting invisible link: {link}")
+                        if all(tag.name == 'br' for tag in link.contents):
+                            link.replace_with(
+                                BeautifulSoup("<br/>", "html.parser")
+                            )
+                        else:
+                            link.decompose()
+
+                    # Save the modified HTML back to page.text
+                    page.content[field] = str(soup)
```

Comment:
I'd replace this with a call to `remove_empty_links` as well and also add the check to see if the content changed before setting it, so you don't emit quite as many `UPDATE`statements, otherwise this looks good to me now.
```suggestion
                cleaned = remove_empty_links(page.content[field])
                if page.content[field] != cleaned:
                    page.content[field] = str(soup)
```